### DISCUSSION TITLE
Barcode scanner API / Expand on NDC Conversion in Postgres

### DISCUSSION THREAD

**Discussion by Bridg109** [coderxio/sagerx#213] [Use cases]
# Proposal
Build out user defined functions for NDC conversion in Postgres so that it will take UPC and Data Matrics scan plus validate against NDC master table

# Rationale
So DB pings from API can take a scanned barcode or entered NDC and return matching NDC

**Comment by jrlegrand** [comment]
We should add support for converting product NDCs to "NDC9" format.
Examples:

"17478-742"
"47781-730"
"76333-265"
"0121-0890"
"63323-010"
